RecoLocalTracker/SiPixelRecHits/plugins/PixelCPEGenericESProducer.cc

```diff
@@ -105,6 +105,8 @@ void PixelCPEGenericESProducer::fillDescriptions(edm::ConfigurationDescriptions&
   // specific to PixelCPEGenericESProducer
   desc.add<std::string>("ComponentName", "PixelCPEGeneric");
   desc.add<edm::ESInputTag>("MagneticFieldRecord", edm::ESInputTag(""));
+  desc.addOptional<bool>("useLAAlignmentOffsets", false)->setComment("deprecated");
```

Comment:
@ianna why this change?

Reply:
I think rebase didn't take into an account reverted changes:
```

Revert "Update mtdParameters for D49 and D60 (Bug fix)", This reverts…  … 1307d75

Revert "[backport] MTD geometry and reconstruction: synchronise with …  … faf18ac

change ECAL pedestals requireStableBeam flag for MWGR test        2a9cbcd

Revert "Revert "[backport] MTD geometry and reconstruction: synchroni…  …f2a994f

revert changes breaking online HLT menu in production release         93afbed
```